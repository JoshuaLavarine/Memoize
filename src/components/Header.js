import React from "react";

const Header = () => {
  return (
    <div className="Header">
      <h1>The Software Pioneers</h1>
      <h2>Fixing bugs since September 9th, 1945</h2>
    </div>
  );
};

export default Header;