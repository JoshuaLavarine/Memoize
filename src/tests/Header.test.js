import React from "react";
import Header from "../Components/Header";
import { shallow } from "enzyme";

describe("Header", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
			<Header 
		/>    
    );
  });
	
  it("should match the snapshot with all data passed in", () => {
    expect(wrapper).toMatchSnapshot();
  });
});