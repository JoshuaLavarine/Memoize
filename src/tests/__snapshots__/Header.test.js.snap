// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all data passed in 1`] = `
<div
  className="Header"
>
  <h1>
    The Software Pioneers
  </h1>
  <h2>
    Fixing bugs since September 9th, 1945
  </h2>
</div>
`;
